contacts-rails/app/views/contacts/edit.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="jumbotron">
+    <h1>EDIT CONTACT</h1>
+</div>
+<div class="container">
```

Comment:
This might not have been covered in the class yet as it's a little bit more [advanced](https://guides.rubyonrails.org/form_helpers.html#binding-a-form-to-an-object) but I think it would be good if you get exposed to it in advance so you can see the power of rails, again _don't_ worry if you can't follow what's going on here, the takeaway is that rails will provide ways to simplify the creation of forms like these. For example, this form can be converted to look like this:
```erb
<div class="container">
  <%= form_for @contact, url: { action: 'update' } do |f| %>
      <div class="form-group">
        <%= f.text_field :first_name, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.text_field :last_name, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.text_field :phone, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.phone_field :email, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.text_field :address, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.text_field :city, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.text_field :state, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.text_field :zip, class: 'form-control' %>
      </div>
      <div class="form-group">
        <%= f.submit 'Save', class: 'btn btn-primary btn-sm form-control' %>
      </div>
  <% end %>
</div>
```